src/py42/clients/cases.py

```diff
@@ -0,0 +1,175 @@
+from enum import Enum
+
+from py42.util import parse_timestamp_to_milliseconds_precision
+
+
+class CaseStatus(Enum):
+    OPEN = u"OPEN"
+    CLOSED = u"CLOSED"
+
+
+class CasesClient(object):
+    """A client to expose cases API.
+
+    `Rest documentation <https://developer.code42.com/api/#tag/Cases>`__
+    """
+
+    def __init__(self, cases_service, cases_file_event_service):
+        self._cases_service = cases_service
+        self._file_events = cases_file_event_service
+
+    @property
+    def file_events(self):
+        """A collection of methods for managing file events associated with a given case.
+
+        Returns :
+            :class:`py42.services.casesfileevents.FileEventsService`
+        """
+        return self._file_events
+
+    def create(
+        self, name, subject=None, assignee=None, description=None, findings=None
+    ):
+        """Creates a new case.
+
+        Args:
+            name (str): Name of the case.
+            subject (str, optional): User UID of a subject of a case.
+            assignee (str, optional): User UID of the assignee.
+            description (str, optional): Description of the case
+            findings (str, optional): Observations of the case.
+
+        Returns
+            :class:`py42.response.Py42Response`
+        """
+        return self._cases_service.create(
+            name,
+            subject=subject,
+            assignee=assignee,
+            description=description,
+            findings=findings,
+        )
+
+    @staticmethod
+    def _make_range(begin_time, end_time):
+        if not begin_time or not end_time:
+            return None
+        end = parse_timestamp_to_milliseconds_precision(end_time)
+        start = parse_timestamp_to_milliseconds_precision(begin_time)
+        return "{}/{}".format(start, end)
+
+    def get_all(
+        self,
+        name=None,
+        status=None,
+        created_at_begin_time=None,
+        created_at_end_time=None,
+        updated_at_begin_time=None,
+        updated_at_end_time=None,
+        subject=None,
+        assignee=None,
+        page_number=1,
+        page_size=100,
+        sort_direction=u"asc",
+        sort_key=u"number",
+        **kwargs
+    ):
+        """Gets all cases.
+
+        Args:
+            name (str, optional): Filter results by case name, matches partial names. Defaults to None.
+            status (enum, optional): Filter results by case status. Defaults to None.
```

Comment:
The type should be `str`, not `enum`

Reply:
Sorry for the misunderstand, I just meant use the enums like we do in py42 in other places, not importh this python package

Reply:
Added.